Apps/Immunization/src/Models/ImmunizationModel.cs

```diff
@@ -23,6 +23,11 @@ namespace HealthGateway.Immunization.Models
     /// </summary>
     public class ImmunizationModel
     {
+        /// <summary>
+        /// Gets or sets the Immunization id.
+        /// </summary>
+        public string Id { get; set; } = string.Empty;
```

Comment:
I think Service code needs to come up with a logic to generate an Id for ImmunizationModel by using GuidGeneration or concatenation of "Location + Immunized (date)".

Reply:
It has to be persistent, otherwise the comments will not work. I will check if the Id comes with the payload